calico-vpp-agent/routing/routing_server.go

```diff
@@ -214,26 +240,41 @@ func (s *Server) getDefaultBGPConfig(log *logrus.Entry, clientv3 calicov3cli.Int
 			}
 			conf.Spec.ASNumber = &asn
 		}
+		if conf.Spec.ListenPort == 0 {
+			conf.Spec.ListenPort = 179
+		}
+		if conf.Spec.LogSeverityScreen == "" {
+			conf.Spec.LogSeverityScreen = "Info"
+		}
 		return &conf.Spec, nil
 	}
 	switch err.(type) {
 	case calicoerr.ErrorResourceDoesNotExist:
 		log.Debug("No default BGP config found, using default options")
 		ret := &calicov3.BGPConfigurationSpec{
-			LogSeverityScreen:     "INFO",
+			LogSeverityScreen:     "Info",
 			NodeToNodeMeshEnabled: &b,
 		}
 		asn, err := numorstring.ASNumberFromString("64512")
 		if err != nil {
 			return nil, err
 		}
 		ret.ASNumber = &asn
+		ret.ListenPort = 179
```

Comment:
This should go in the struct initialization (l.255), asn is out only because we need to call ASNumberFromString